sql/src/main/java/io/crate/operation/scalar/cast/CastFunctionResolver.java

```diff
@@ -66,4 +70,16 @@ public static FunctionInfo functionInfo(DataType dataType, DataType returnType)
         }
         return new FunctionInfo(new FunctionIdent(functionName, ImmutableList.of(dataType)), returnType);
     }
+
+    @Nullable
+    public static Symbol castSymbol(Symbol symbol, DataType targetType) {
```

Comment:
this is pretty similar to ``cast(..)`` in the ExpressionAnalyzer. Can one or the other be used instead?